docs/integrations/github_actions/aws_oidc.md

```diff
@@ -0,0 +1,157 @@
+Steampipe will load these short-lived [credentials from Environment variables](https://hub.steampipe.io/plugins/turbot/aws#credentials-from-environment-variables) to run the benchmark.
+
+Before running the compliance benchmark, create a new folder on the branch specified in your GitHub repository to save the benchmark output. In our example, we will save the outputs to the folder `steampipe/benchmarks/aws`. `GITHUB_WORKSPACE` is an environment variable which refers to the default working directory on the runner for steps, and the default location of your repository when using the [checkout](https://github.com/actions/checkout) action.
+
+Here's the step that runs the [AWS Compliance](https://hub.steampipe.io/mods/turbot/aws_compliance) mod.
+
+```yaml
+- name: "Run Steampipe benchmark"
+  id: steampipe-benchmark
+  run: |
+
+    # Install the Steampipe AWS Compliance mod
+    steampipe mod install github.com/turbot/steampipe-mod-aws-compliance 
+    cd .steampipe/mods/github.com/turbot/steampipe-mod-aws-compliance*
+    # Run the AWS CIS v1.5.0 benchmark
+    steampipe check benchmark.cis_v150 --export=${GITHUB_WORKSPACE}/steampipe/benchmarks/aws/cis_v150_"$(date +"%d_%B_%Y")".html --output=none
+```
+
+Here's the step that pushes the output of the benchmark to your repository. Update the `working-directory` to the folder created in the above step. This should be the same location used in the above `--export` argument.
+
+```yaml
+- name: "Commit the file to github"
+  id: push-to-gh
+  working-directory: steampipe/benchmarks/aws
+  run: |
+
+    git config user.name github-actions
+    git config user.email github-actions@github.com
+    git add cis_v150_"$(date +"%d_%B_%Y")".html 
+    git commit -m "Add Steampipe Benchmark Results"
+    git push
+```
+
+## GitHub's OIDC provider for AWS
+
+To try this yourself you'll need to set up OIDC for your cloud provider. GitHub's [Security hardening your deployments](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments) page has instructions for using OpenID Connect with various providers. To help you follow those instructions we have created a [Terraform sample](https://github.com/turbot/steampipe-samples/tree/main/all/github-actions-oidc/aws). If you prefer AWS CloudFormation, you can make use of this [link](https://github.com/aws-actions/configure-aws-credentials#sample-iam-role-cloudformation-template) to get started. This guide will demonstrate the Terraform implementation. This Terraform script will create two AWS resources, an Identity provider and IAM Role in your account. These resources together form an OIDC trust between the AWS IAM role and your GitHub workflow(s) that need access to the cloud.
```

Comment:
I'd replace the first sentenence with something like:

`In order for AWS to trust GitHub, you must configure OIDC as an identity provider in your AWS account. `